modules/pool/invest/lib/useInvest.ts

```diff
@@ -31,6 +32,17 @@ export function useInvest() {
         amount: getUserBalanceForToken(token.address),
     }));
 
+    const hasValidUserInput =
+        !selectedInvestTokensWithAmounts.every((token) => parseFloat(token.amount) === 0) &&
+        selectedInvestTokensWithAmounts.every((token) => {
+            console.log(
```

Comment:
remove this console.log plz

Reply:
my bad (i might need a local lint rule for console.log)